CONTRIBUTING.md

```diff
@@ -22,18 +23,24 @@ If you'd like to contribute code, here are some things to keep in mind.
   If you have questions, feel free to mail the mailing list.
 * Add your new tests to get auto-run by Travis in the .travis.yml to make sure that
   other people don't accidentally break your module.
-* Check out our Coding Standards in the [docs/ directory](https://github.com/aces/Loris/tree/master/docs) and also our [Code Review Checklist](https://github.com/aces/Loris/wiki/Code-Review-Checklist) in the [GitHub Wiki](https://github.com/aces/Loris/wiki) and the Pull Request guidelines Readme in this directory. 
+
 * Try and make sure you run PHP codesniffer using the standards file in
   docs/LorisCS.xml before sending any pull request, otherwise the Loris tests may
   fail and we won't be able to merge your pull request.
-* Keep in mind that people are currently using Loris so try and make any changes
+  
+
+  * Check out our Coding Standards in the [docs/ directory](https://github.com/aces/Loris/tree/master/docs) and also our [Code Review Checklist](https://github.com/aces/Loris/wiki/Code-Review-Checklist) in the [GitHub Wiki](https://github.com/aces/Loris/wiki) and the Pull Request guidelines Readme in this directory. 
+  
+* Keep in mind that many projects using Loris so try to make any changes
   backwards-compatible with existing installations. If you must change something
   in a non-backwards-compatible way, document it in your pull request description and
   tag it with "Caveat For Existing Projects" so that we know that the change needs
-  to be mentioned in release notes. Non-backwards-compatible changes should be sent
-  to the next major release (e.g. 17.X -> 18.0) while backwards-compatible changes should
-  be sent to the next minor release (e.g. 17.1.0 -> 17.2.0) and backwards-compatible bug
-  fixes should be sent to the next minor release update (e.g. 17.1.0 -> 17.1.1)
+  to be mentioned in release notes. 
+* Choosing a branch: see [Contributing to the Code](https://github.com/aces/Loris/wiki/Contributing-to-the-Code) for full guidelines. 
+  Non-backwards-compatible changes should be sent
+  to the `major` branch (e.g. 18.* -> 19.0), while backwards-compatible changes should
+  be sent to the `minor` branch (e.g. 18.0.* -> 18.1), and backwards-compatible bug
+  fixes should be sent to `bugfix` branch (e.g. 18.1.0 -> 18.1.1)
```

Comment:
It would make more sense to me to have the brief summary, and then "see ... for full guidelines" than the other way around, but I don't feel passionately about it either.